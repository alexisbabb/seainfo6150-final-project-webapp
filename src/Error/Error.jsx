import React from 'react'; 
import styles from "./Error.module.css";

const Error = () => {
    return (
        <article className = {styles.pageContent}>
            <header className = {styles.pageTitle} >
            <p>Oh No! It looks like this page isn't valid. Sorry about that!</p>
            <img className = {styles.pageImage} src="https://cdn3.volusion.com/euhfr.xvuyx/v/vspfiles/photos/EG1155-1374-2.jpg?v-cache=1325487014" alt="Error"/>
            </header>
        </article>
    )
}

export default Error
