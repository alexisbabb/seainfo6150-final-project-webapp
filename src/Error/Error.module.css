.pageContent{
    background-image: linear-gradient(to bottom, rgba(255, 255, 0, 0.25), rgba(255, 0, 0, 0.25));
    font-family: "Helvetica", sans-serif; 
}

.pageTitle {
    font-size: 3em; 
    font-weight: bold;  
    text-align: center; 
    padding: 0.5em; 
}

.pageImage{
    width: 35%; 
    border-radius: 10%; 
    display: block; 
    margin-left: auto; 
    margin-right: auto; 
}

