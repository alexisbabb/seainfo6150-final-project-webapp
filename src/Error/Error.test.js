import React from "react";
import Error from "./Error"

describe("Error tests", () => {
    it("renders correctly", () => {
        const src = "https://cdn3.volusion.com/euhfr.xvuyx/v/vspfiles/photos/EG1155-1374-2.jpg?v-cache=1325487014";
        const { container } = render(<Error src={src}/>);
        expect(container).toMatchSnapshot();
    });
});