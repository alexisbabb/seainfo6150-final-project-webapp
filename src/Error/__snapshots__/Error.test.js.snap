// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error tests renders correctly 1`] = `
<div>
  <article
    class="pageContent"
  >
    <header
      class="pageTitle"
    >
      <p>
        Oh No! It looks like this page isn't valid. Sorry about that!
      </p>
      <img
        alt="Error"
        class="pageImage"
        src="https://cdn3.volusion.com/euhfr.xvuyx/v/vspfiles/photos/EG1155-1374-2.jpg?v-cache=1325487014"
      />
    </header>
  </article>
</div>
`;
